tools/dgeni/processors/categorizer.ts

```diff
@@ -88,6 +90,12 @@ export class Categorizer implements Processor {
     classDoc.methods.sort(sortCategorizedMembers);
     classDoc.properties.sort(sortCategorizedMembers);
 
+    // Filter the extends clauses for clauses with an associated Dgeni document. Clauses without
+    // a document are unknown and should not be mentioned in the documentation for this class.
+    classDoc.extendedDocs = classDoc.extendsClauses
+      .filter(clause => clause.doc)
+      .map(clause => clause.doc!);
```

Comment:
Did you confirm that this filters out mixin classes like `MatButtonBase`?

Reply:
Yeah, those are filtered out because there is no Dgeni doc for those.